test/test_sort_and_select.py

```diff
@@ -119,6 +119,12 @@ def test_sort(self, device):
             # Test that we still have proper sorting with duplicate keys
             self.assertIsOrdered('descending', x, res2val, res2ind, 'random with duplicate keys')
 
+            # Test argument sorting with and without stable
+            x = torch.tensor([1, 10, 2, 2, 3, 7, 7, 8, 9, 9] * 3)
+            self.assertEqual(torch.argsort(x, stable=True), torch.sort(x, stable=True).indices)
+            self.assertEqual(torch.argsort(x, stable=False), torch.sort(x, stable=False).indices)
```

Comment:
could you please include without `stable` specified? To test that it complies with `sort`, i.e. it is not stable.

Reply:
Sure, done :)